PowerShell/ScubaGear/Testing/Unit/Rego/AAD/AADBaseConfig.rego

```diff
@@ -0,0 +1,266 @@
+package aad_test
+import rego.v1
+import data.utils.aad.INT_MAX
+
+ConditionalAccessPolicies := {
+    "Conditions": {
+        "Applications": {
+            "IncludeApplications": [
+                "All"
+            ],
+            "ExcludeApplications": [],
+            "IncludeUserActions": [
+                "urn:user:registersecurityinfo"
+            ]
+        },
+        "Users": {
+            "IncludeUsers": [
+                "All"
+            ],
+            "ExcludeUsers": [],
+            "ExcludeGroups": [],
+            "ExcludeRoles": []
+        },
+        "UserRiskLevels": [
+            "high"
+        ],
+        "SignInRiskLevels": [
+            "high"
+        ],
```

Comment:
the default config is the values needed to produce a pass in the test rule. So UserRiskLevels with array value as high is needed for on of the policies to produce a pass. I can extrapolate them out? We will just be moving away from our base model, because we would only be able to add the values using json patch.